packages/kotti-ui/source/kotti-actionbar/components/ActionbarMenu.vue

```diff
@@ -1,20 +1,20 @@
 <template>
-	<div class="actionbar-nav">
+	<div class="kt-actionbar-nav">
 		<!-- FIXME: External Dependencies Should not Be Used without Injection -->
 		<router-link
 			v-for="(item, index) in menu"
 			:key="index"
-			class="actionbar-nav__item"
+			class="kt-actionbar-nav__item"
 			:class="navItemClass(item)"
 			tag="li"
 			:to="item.to"
 		>
 			<i
-				class="yoco actionbar-nav__item__icon"
+				class="yoco kt-actionbar-nav__item__icon"
```

Comment:
i think this is now `kt-actionbar-nav__icon` . looks wrong

Reply:
```suggestion
				class="yoco kt-actionbar-nav__icon"
```